src/shared/static/static-service.interface.ts

```diff
@@ -0,0 +1,16 @@
+export interface StaticService {
+  // Below are service specific functions
+  saveImage(type: 'screenshot' | 'diff' | 'baseline', imageBuffer: Buffer);
+  getImage(fileName: string);
+  deleteImage(imageName: string);
+
+  doesFileExistLocally(fileName: string);
```

Comment:
seems like related only to s3 implementation 
I guess we should do more generic interface with methods like 
```js
  saveImage(type: 'screenshot' | 'diff' | 'baseline', imageBuffer: Buffer);
  getImage(fileName: string);
  deleteImage(imageName: string);
```